storybook/src/docs/typeface.docs.mdx

```diff
@@ -2,16 +2,16 @@
 
 import { Meta, Typeset } from "@storybook/blocks";
 
-<Meta title="Docs/Assets/Font" />
+<Meta title="Foundation/Assets/Typeface" />
 
-# Font
+# Typeface
```

Comment:
'typeface' is the correct name, but it is less commonly used than 'font'. Should we add 'font' in parentheses after 'typeface'?

Reply:
No, I don’t think so. We should pick one, not make a grey area. I could add two sentences on the difference between the two on the page itself?

Come to think of it, we could / should rename ‘Typography’ to ‘Font’ in the ‘Design Tokens’ section.

Reply:
In that case, I think convention beats correctness in this case. I'm pretty sure people generally use 'font' instead of 'typeface', so we should do the same. [Just as google does](https://fonts.google.com/) ;)

Reply:
You mean that site where you evaluate certain combinations of a typeface with a font size, weight, and style? Shortly, fonts?

And yes, they use the word ‘typeface’ correctly as well 😁

<img width="492" alt="Screenshot 2024-07-16 at 16 02 27" src="https://github.com/user-attachments/assets/6cb5ee00-c245-4f05-b846-8ef3432b1039">

Reply:
Done.